src/main/java/duke/command/ViewDayCommand.java

```diff
@@ -0,0 +1,29 @@
+package duke.command;

```

Comment:
I understand this is not a requirement, but perhaps should still have javadocs for it?